galsim/optics.py

```diff
@@ -50,6 +50,15 @@ class OpticalPSF(GSObject):
     """A class describing aberrated PSFs due to telescope optics.  Its underlying implementation
     uses an InterpolatedImage to characterize the profile.
 
+    The diffraction effects are characterized by the diffraction angle, which is a function of the
+    ratio lambda / D, where lambda is the wavelength of the light and D is the diameter of the
+    telescope.  Users can specify this diffraction angle in one of two ways.  The first option is to
+    specify the ratio as `lam_over_diam` in whatever arbitrary units they choose (and then
+    self-consistently specify the image scale in the same units).  The second option is to specify
+    the wavelength as `lam` in units of nanometers and the telescope diameter as `diam` in units of
+    meters.  OpticalPSF will then convert the ratio to arcsec, and image scales must be specified in
+    arcsec as well.
```

Comment:
I think you mean convert the ratio to `scale_units` and image scales must be specified in `scale_units` as well.